pose-detection/src/blazepose_mediapipe/README.md

```diff
@@ -140,3 +141,16 @@ There is a difference of how things are loaded between the two runtimes. For the
 | Lite model | 10.41MB | 1.91s |
 | Full model | 13.8MB | 1.91s |
 | Heavy model | 34.7MB | 4.82s |
+
+## Model Accuracy
+This table compares MediaPipe BlazePose accuracy against other state-of-the-art
+pose solutions. For more detail, see the [article](https://google.github.io/mediapipe/solutions/pose#pose-estimation-quality).
+| Method | Yoga | Yoga | Dance | Dance | HIIT | HIIT |
+| --- | --- | --- | --- | --- | --- | --- |
```

Comment:
Please add links to PCK@0.2 and mAP definitions. Original table refers them. For mAP https://cocodataset.org/#keypoints-eval . For PCK@0.2 https://github.com/cbsudux/Human-Pose-Estimation-101#percentage-of-correct-key-points---pck